uber/config.py

```diff
@@ -160,14 +167,29 @@ def PREREG_DONATION_TIERS(self):
         return dict(self.PREREG_DONATION_OPTS)
 
     @property
-    def SUPPORTERS_ENABLED(self):
-        return self.SUPPORTER_LEVEL in self.PREREG_DONATION_TIERS
-
-    @property
```

Comment:
Ditto for this property; it's not actually used in either the code plugin or our event-specific plugin.